src/Components/GraduatesList/GraduatesList.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from "react";
+import { Checkbox, Input, Table } from "semantic-ui-react";
+import FilterButton from "./FilterButton";
+import Graduated from "./Graduated";
+import axios from "axios";
+
+
+class GraduatesList extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			graduates: [],
+			filterBy: 'All',
+			filterCriteria:'All'
+		};
+	}
+
+	componentDidMount() {
+		const API_URL = process.env.REACT_APP_API_URL;
+	  axios
+			.get(`${API_URL}/empresas-egresades`)
+			.then(response => {
+				const graduates = response.data;
+				this.setState({ graduates });
+			})
+			.catch(function(error) {
+				console.log(error);
+		});
+	}
+
+	listGraduates=() => {
+		if (this.state.filterCriteria === 'All')
+			return this.MapGraduates(this.state.graduates);
+		else
+			return this.MapGraduates(this.factoryFilter())
+	}
+	
+	MapGraduates(listEgresades){
+		return(
+			listEgresades.map((currentList, i) => {
+				return <Graduated item={currentList} key={i} />
+		}));
+	}
+
+	factoryFilter(){
+    if(this.state.filterBy === 'EnglishLevel')
+			return this.filterByEnglishLevel(this.state.filterCriteria);
+	}
+
+	
+	filterByEnglishLevel(filterCriteria){
+		const ListFiltered = [];
+		this.state.graduates.forEach((graduated) => {	
```

Comment:
Guys for this part you can use filter method from javascript, you don't have to ask about an object inside another object  like module completed filter :D